tests/Tests/700__sandbox/701__performance_sandbox.robot

```diff
@@ -84,5 +85,8 @@ Performance Suite Setup
     [Documentation]    Disable CopiedCSVs in OLMConfig to not watch csv created in every namespace
     ...   since copied CSVs consume an untenable amount of resources, such as OLM’s memory usage,
     ...   cluster etcd limits, and networking
-    Oc Apply    kind=OLMConfig    src=tests/Tests/700__sandbox/olm.yaml
-    Run    git clone https://github.com/codeready-toolchain/toolchain-e2e.git
+     Oc Apply    kind=OLMConfig    src=tests/Tests/700__sandbox/olm.yaml
+     Run    git clone https://github.com/codeready-toolchain/toolchain-e2e.git
+     FOR    ${isv}    IN    @{ISV_DATA.values()}
+           Install ISV By Name    ${isv[0]}      ${isv[1]}
```

Comment:
don't you need the CRs to be deployed for each ISVs?

Reply:
NOPE